pkg/stub/status.go

```diff
@@ -52,7 +52,32 @@ func (h *Handler) syncOperatorStatus() {
 	oldConditions := co.Status.Conditions
 	co.Status.Conditions = computeStatusConditions(oldConditions, ns,
 		dnses, daemonsets)
-	co.Status.Version = operatorversion.Version
+	co.Status.RelatedObjects = []configv1.ObjectReference{
+		{
+			Resource: "namespaces",
+			Name:     "openshift-dns-operator",
+		},
+		{
+			Resource: "namespaces",
+			Name:     ns.Name,
+		},
+	}
+
+	oldVersions := co.Status.Versions
+	coreDNSVersion := h.Config.CoreDNSImage
+	if versionIdx := strings.Index(coreDNSVersion, ":"); versionIdx >= 0 {
+		coreDNSVersion = coreDNSVersion[versionIdx+1:]
```

Comment:
Instead of extracting the coredns revision, do we want to use the coredns imageID ?
(Similar to https://github.com/openshift/cluster-version-operator/pull/104/files#diff-437d518ae5ea84d0c4284dd06bfe3fd7R79 )

Reply:
Will that be similar to 'docker.io/openshift/origin-coredns:v4.0'  (it may have openshift registry prefix instead of docker.io)?

Reply:
In the last test run, it was `registry.svc.ci.openshift.org/ci-op-slzhbpgm/stable@sha256:e4936a702d7d466a64a6a9359f35c7ad528bba7c35fe5c582a90e46f9051d8b8`.

Reply:
If we use this as coredns version, how can we decrypt which version of coredns is used by user?
It does not look like `e4936a702d7d466a6...` maps to openshift/coredns commit revision.

Reply:
Yeah, I'm pretty sure that's a checksum of the container image, and I don't know how to translate it back to a Git commit or other meaningful identifier.

Reply:
There's some oc tooling to find all this info like `oc adm release info --pullspecs` and `oc image info`... probably more.

Reply:
Are you saying we could look at said tooling to see how the operator can find this info, or are you saying we are providing sufficient info for a human to use the tooling to find the needed info?

Reply:
For now the latter. Let's report the thing we're actually deploying and if other details need to be derived from it, approach separately.